fuseutil/not_implemented_file_system.go

```diff
@@ -138,6 +138,12 @@ func (fs *NotImplementedFileSystem) ReadFile(
 	return fuse.ENOSYS
 }
 
+func (fs *NotImplementedFileSystem) VectoredRead(
```

Comment:
I know you’re just copying existing style, but here and elsewhere, can you please remove all line breaks from function signatures for newly added code? They should just be one line going forward.

Reply:
I did it for readbenchfs, but not for new functions surrounded with the old code-style functions - I think a single function in a different style may look bad... Maybe we'll just change style in a separate commit and then change the style for newer commits?